src/test/java/org/jenkinsci/plugins/workflow/log/LogStorageTestBase.java

```diff
@@ -149,11 +148,6 @@ protected static void close(TaskListener listener) throws Exception {
         VirtualChannel channel = r.createOnlineSlave().getChannel();
         channel.call(new RemotePrint("overall from agent", overall));
         channel.call(new RemotePrint("step from agent", step));
-        while (!IOUtils.toString(text().readAll()).contains("overall from agent") || !IOUtils.toString(text().readAll()).contains("step from agent")) {
-            // TODO current cloud implementations may be unable to honor the completed flag on remotely printed messages, pending some way to have all affected loggers confirm they have flushed
```

Comment:
Plan to address this differently, by having the cloud implementation honor `flush`.

Reply:
(which I did)